pom.xml

```diff
@@ -981,7 +981,9 @@
       --add-opens java.base/java.util=ALL-UNNAMED
       --add-opens java.base/java.util.concurrent=ALL-UNNAMED
       --add-exports java.base/jdk.internal.misc=ALL-UNNAMED
-      --add-exports java.security.jgss/sun.security.krb5=ALL-UNNAMED</hbase-surefire.jdk11.flags>
+      --add-exports java.security.jgss/sun.security.krb5=ALL-UNNAMED
+      --add-exports java.base/sun.net.dns=ALL-UNNAMED
```

Comment:
Hi @stoty, jira mentions build is failing for jdk 17, so shouldn't this go under `hbase-surefire.jdk17.flags`? Or is it needed for jdk11 as well?
Also in https://github.com/apache/hadoop/pull/6118/files it is under jdk17

Reply:
The packages are present in JDK 11.
It seems that these modules were added JDK 11, but are only enforced in JDK17.
I have confirmed that JDK11 accepts these parameters, and does not print warnings.

AFAIU If we added these for JDK17 only, that would solve the JDK startup failure problem shown in the stack trace, but JDK11 would still print warnings when we execute those Hadoop methods.

So I believe that the correct place to add them is JDK11.

Reply:
Thank you for the explanation.